### ISSUE TITLE
Re/Uninstall looks for displaylink-installer.sh

### ISSUE THREAD

**Issue by robotard** [AdnanHodzic/displaylink-debian#376]
I've recently seen some issue on reinstalling ( I think uninstall too ) whereby it's looking for the following path:

/opt/displaylink/displaylink-installer.sh

Strangely if you copy the contents from the 5.2 folder there, and uninstall, it clears up this folder, but am guessing somewhere in the process this is not getting copied across... Not sure under which circumstances it's not copied as obvs am running as sudo, but have seen this occuring,,,

**Comment by robotard** [comment]
```
Platform requirements satisfied, proceeding ...

-------------------------------------------------------------------

Uninstalling ...

bash: /opt/displaylink/displaylink-installer.sh: No such file or directory

-------------------------------------------------------------------

Performing clean-up
```

**Comment by AdnanHodzic** [comment]
I believe this was fixed as part of your #377 PR

**Comment by robotard** [comment]
It does appear so. :) 

Not entirely sure as to why that my little fix cascaded down to that other one being fixed...
